### DISCUSSION TITLE
Release v1.5.0 (beta 2)

### DISCUSSION THREAD

**Discussion by RevZero** [X2CommunityCore/xcom2-launcher#279] [-Releases-]
**Download:** [XCOM2_AlternativeModLauncher_1.5.0-beta2.zip](https://github.com/X2CommunityCore/xcom2-launcher/releases/download/v1.5.0-beta.2/XCOM2_AlternativeModLauncher_1.5.0-beta2.zip) 
> Note that this is a beta-release and may contain bugs. Consider waiting for the final release to get a more polished experience.

## Preface

This is a important hotfix following the first 1.5.0 beta, because **enabling/disabling mods wasn't working properly** in that version. 

## Infos

Refer to the [installation instructions](https://github.com/X2CommunityCore/xcom2-launcher/wiki/Installation) about how to install or update AML.

- Check out [the Wiki](https://github.com/X2CommunityCore/xcom2-launcher/wiki) for some useful tips.
- For questions or suggestions you are welcome to join our [Discord channel](https://discord.gg/QHSVGRn) or start a [Discussion](https://github.com/X2CommunityCore/xcom2-launcher/discussions).
- If AML is crashing or otherwise not working as expected, you can [report a bug here](https://github.com/X2CommunityCore/xcom2-launcher/issues/new/choose).

## Changelog

- Fixed writing XCOM 2 `XComModOptions.ini` to wrong the folder, causing changes from mods not being applied to the game.
- Fixed "Quick toggle arguments" option not being visible in the settings dialog.
- Fixed spelling error `File->Open Special Folder->XCOM 2 - Iinstallation".

[View full v1.5.0 changelog](https://github.com/X2CommunityCore/xcom2-launcher/releases/tag/v1.5.0-beta).

<hr /><em>This discussion was created from the release <a href='https://github.com/X2CommunityCore/xcom2-launcher/releases/tag/v1.5.0-beta.2'>Release v1.5.0 (beta 2)</a>.</em>

**Comment by Dragon-32** [comment]
I've used this a fair bit since September without spotting any issues. Are there plans to move this to be the full, release version?

**Reply by RevZero** [reply]
Yes, but I want to address #280 first and have it in this release because its related to another recent fix. Depending on the scope of changes, maybe I will have another (short) beta release to be on the safe side.

**Reply by Dragon-32** [reply]
Oh, OK. Thanks for the reply. For me at least beta 2 is working really well :)
